docs/README.md

```diff
@@ -16,19 +16,22 @@ This github repository contains documentation and API usage examples that can be
 
 ## API usage examples and Scripts
 
-### Shell/Curl Examples
-
-* [How to generate auth tokens](./api-tokens.md)
-
 ### Maximo Asset Import
 
 * [Importing Maximo Locations into Weather Operations Center](./maximo-integration-asset-import.md)
 
-### Custom Geospatial Queries
+### Geospatial Analytics
+
+#### API Authentication
+
+* [How to generate and use access tokens](./geospatial-api.md#authentication)
+
+#### Custom Geospatial Queries
 
 * [Performing Custom Geospatial Queries, Registration and Visualization](./custom-geospatial-query-extension.md)
 
 ### Agriculture Add-on Specific Examples and Samples
+* [How to generate auth tokens](./api-tokens.md)
```

Comment:
The auth token how to might be a bit hidden now.

Perhaps a more general TOC heading might be better with each component listed.